src/slur.cpp

```diff
@@ -619,6 +625,72 @@ void Slur::RebalanceShifts(int &shiftLeft, int &shiftRight, const double distanc
     }
 }
 
+void Slur::AdjustSlurFromBulge(FloatingCurvePositioner *curve, BezierCurve &bezierCurve, const int unit)
+{
+    if (bezierCurve.p1.x >= bezierCurve.p2.x) return;
+
+    // Read the bulge attribute
+    Att converter;
+    data_BULGE bulge = converter.StrToBulge(this->GetBulge());
```

Comment:
Instead of doing this here it should be at the input (and output). You need to change the libMEI `config.yml` in order to use the typedef `data_BULGE` you added for `@bulge`. Let me know if you need assistance for this

Reply:
Right, I've [opened a PR](https://github.com/rism-digital/libmei/pull/8) on LibMEI.